lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/components/RuntimeVersionManager.scala

```diff
@@ -339,7 +339,7 @@ class RuntimeVersionManager(
     result match {
       case (path, Failure(exception)) =>
         logger.warn(
-          "{} at [{}] has been skipped due to the following error: {}",
```

Comment:
Why not print this error?

Reply:
it will actually skip it because of the way the last argument is treated if it is an exception. So it would always be empty from what I saw.